src/cloud/cdn/configure-fastly.md

```diff
@@ -123,41 +122,126 @@ To upload the Fastly VCL:
 
 1. After the upload completes, refresh the cache according to the notification at the top of the page.
 
-## DNS configuration {#dns}
+## Provision SSL/TLS certificates
+
+Magento provides a Domain-Validated Let's Encrypt SSL/TLS certificate to serve secure HTTPS traffic from Fastly. Magento provides one certificate for each Pro Production, Staging, and Starter Production environment to secure all domains in that environment.
+
+{:.bs-callout-info}
+You can provide your own TLS/SSL certificate instead of using the Let's Encrypt certificate provided by Magento. However, this process requires additional work to setup and maintain. To choose this option, submit a [Magento Support ticket](https://support.magento.com/hc/en-us/articles/360019088251) or work with your CTA to add custom hosted certificates to your Cloud environments.
```

Comment:
```suggestion
You can provide your own TLS/SSL certificate instead of using the Let's Encrypt certificate provided by Magento. However, this process requires additional work to set up and maintain. To choose this option, submit a [Magento Support ticket](https://support.magento.com/hc/en-us/articles/360019088251) or work with your CTA to add custom hosted certificates to your Cloud environments.
```